src/Columns/ColumnAggregateFunction.h

```diff
@@ -26,6 +17,19 @@ using ArenaPtr = std::shared_ptr<Arena>;
 using ConstArenaPtr = std::shared_ptr<const Arena>;
 using ConstArenas = std::vector<ConstArenaPtr>;
 
+using AggregateDataPtr = char *;
+using AggregateDataPtrs = std::vector<AggregateDataPtr>;
+using ConstAggregateDataPtr = const char *;
+
```

Comment:
I support the changes you are doing, but I'm not sure about places like this where it's unclear that `AggregateDataPtr = char *`.

Maybe we should prefer creating `_fwd.h` files like for `Context` in those cases?

Reply:
Makes sense :ok_hand: